docs/getting-started-step-by-step/02a-managed-resources-aws.md

```diff
@@ -0,0 +1,135 @@
+# Using Managed Resources from AWS: Connect Crossplane to manage AWS resources.
+
+In this part, our goal is to configure Crossplane so we can launch and manage AWS resources directly from within Kubernetes. We will leverage the power of the Kubernetes machinery that continuously reconciles the desired state without any manual interaction from users.  
```

Comment:
Showcasing how to use a managed resource directly (i.e. without composition) is also something we've gone back and forth on in the past, and is something we should be careful about. On one hand it's probably best to learn the basic building blocks first - i.e. you learn about Kubernetes Pods before you learn about Deployments. On the other hand (almost) no-one uses Kubernetes by hand-managing Pods. They use higher level constructs like Deployments.

Perhaps a good compromise here would be to call this out explicitly - i.e. that managed resources are the building blocks of Crossplane and that it is not common for folks to use them standalone (and in fact is actively not something we recommend or design for).

Reply:
I've heard a lot of users ask questions about "why they even need composition" in the first place, so I think it's more common than we might realize. Also, we do see a lot of confusion about Composition in general, so I'm assuming many community members are simply using ManagedResources and no Composition (customers are a different story). We also know that many users find Crossplane because they want to use CRDs to manage infrastructure, and they've learned about some of the ideas from projects like config-connector and ASO.

I think calling out the relationship is a good idea though. We might even want to publish the building blocks analogy as we keep using it ourselves.

Reply:
It seems like there are two things here.

1. Strategically, do we want to embrace and optimise for using Crossplane without the composition layer.
2. Pending the outcome of 1, how do we want to frame managed resources.

I'm not convinced that composition is as unapproachable as you might perceive it to be, based on the amount of community engagement (e.g. feature requests, issues, PRs, etc) we see. I agree it's complex and that we should do our utmost to make it approachable, but the same can be said for a lot of popular technologies in our space (e.g. Kubernetes, Terraform, etc).

We've made design decisions that make Crossplane a bit of an odd choice if you're not interested in composition, so there are larger implications than solely documentation if we do choose to embrace and support using Crossplane without composition.

Reply:
@negz: Thanks for pointing this out.

I agree that the focus needs to be on the composition layer. I'm just wondering how to make this approachable. For me, it was important to first understand MR which is why I'm introducing them in a dedicated step. And while MR provide value for themselves we need to optimize for users to adopt XR. 

I did clarify this [here](https://github.com/luebken/crossplane/pull/1/commits/f4ef2754cec285127f569c04bcff7b102746c4a2). But I'm open for other ideas.

Reply:
> I'm not convinced that composition is as unapproachable as you might perceive it to be,...

Ack. I will make sure community feedback is incorporated.